test/app.js

```diff
@@ -129,16 +129,6 @@ describe('Smoke test', () => {
     should(app.reconnect()).be.fulfilled();
   });
 
-  it('throws error when reconnect operation called on unregistered app', async () => {
```

Comment:
Why are we removing this test? it seems the description/summary or the test itself is inaccurate as we are not calling reconnect here.